### PR TITLE
Comment tags sent and version from compile flag

### PR DESCRIPTION
**Pull request by DG12**
just comments

### CODE REVIEW

**Comment by ojousima** [comment]
There seems to be some unrelated issue with SonarCloud, I'll need to sort it out first

File: src/tasks/peripherals/task_nfc.c [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// When NFC reader is in range report" SW: version, MAC: and  ID:DEVICEID"
```
Comment (ojousima) [inline src/tasks/peripherals/task_nfc.c]: The comment should rather be something as in https://github.com/ruuvi/ruuvi.firmware.c/blob/master/src/tasks/actuators/task_led.h
```
/**
 * @file task_led.h
 * @author Otso Jousimaa <otso@ojousima.net>
 * @date 2019-11-18
 * @copyright Ruuvi Innovations Ltd, license BSD-3-Clause.
 *
 * LED control.
 *
 * Typical usage:
 *
 * @code{.c}
 *  ruuvi_driver_status_t err_code = RUUVI_DRIVER_SUCCESS;
 *  err_code = task_gpio_init();
 *  RUUVI_DRIVER_ERROR_CHECK(err_code, RUUVI_DRIVER_SUCCESS;
 *  err_code = task_led_init();
 *  RUUVI_DRIVER_ERROR_CHECK(err_code, RUUVI_DRIVER_SUCCESS;
 *  err_code = task_led_write (RUUVI_BOARD_LED_GREEN, RUUVI_BOARD_LEDS_ACTIVE_STATE);
 *  RUUVI_DRIVER_ERROR_CHECK(err_code, RUUVI_DRIVER_SUCCESS;
 *  err_code = task_led_activity_led_set (RUUVI_BOARD_LED_GREEN);
 *  ruuvi_interface_yield_indication_set (task_led_activity_indicate);
 *  RUUVI_DRIVER_ERROR_CHECK(err_code, RUUVI_DRIVER_SUCCESS;
 *  _do_stuff_();
 *  err_code = task_led_uninit();
 *  RUUVI_DRIVER_ERROR_CHECK(err_code, RUUVI_DRIVER_SUCCESS;
 * @endcode
 */
```

File: src/tasks/peripherals/task_nfc.c [hunk 2]
```diff
@@ -23,7 +24,7 @@ ruuvi_driver_status_t task_nfc_init (void)
     memcpy (version_string, fw_prefix, sizeof (fw_prefix));
     written = snprintf ( (char *) (version_string + sizeof (fw_prefix)),
                          APPLICATION_COMMUNICATION_NFC_TEXT_BUFFER_SIZE - sizeof (fw_prefix),
-                         "%s", APPLICATION_FW_VERSION);
+                         "%s", APPLICATION_FW_VERSION);  // Taken from compile flag. May be MM.nn.rr or git hash
```
Comment (ojousima) [inline src/tasks/peripherals/task_nfc.c]: The actual source is in application_config.h, which happens to pick it from compiler flag or use a default value "RuuviFW DEVEL" as of right now. 
This comment would be better if it pointed to the application_config.h where the macro is documented, if it is documented twice the comments are going to end up out of sync.

File: src/tasks/peripherals/task_nfc.c [hunk 3]
```diff
@@ -143,4 +144,4 @@ ruuvi_driver_status_t task_nfc_init (void)
 {
     return  RUUVI_DRIVER_SUCCESS;
 }
-#endif
\ No newline at end of file
+#endif
```
Comment (ojousima) [inline src/tasks/peripherals/task_nfc.c]: While doing changes, please add a newline to the end of file. It fixes one automatic check complaint.

**Comment by ojousima** [comment]
The Travis error is caused by  https://docs.travis-ci.com/user/pull-requests/#pull-requests-and-security-restrictions and can be ignored

**Comment by DG12** [comment]
Revised comment format.
Should 
#include "ruuvi_interface_dog.h"
be deleted?

**Comment by DG12** [comment]
There seems to be a 4th item sent that is not interpreted by NFC for iPhone app.
See URI type not supported

IMG_1123

File: src/tasks/peripherals/task_nfc.c [hunk 4]
```diff
@@ -7,7 +14,7 @@
 #include "ruuvi_interface_communication_nfc.h"
 #include "ruuvi_interface_communication_radio.h"
 #include "ruuvi_interface_scheduler.h"
-#include "ruuvi_interface_watchdog.h"
+#include "ruuvi_interface_dog.h"
```
Comment (ojousima) [inline src/tasks/peripherals/task_nfc.c]: does this compile?

Comment (DG12) [reply src/tasks/peripherals/task_nfc.c]: Argh! I accidentally lost the "watch" , restored it to my fork, but it seems I neglected to re-pull it .

**Comment by ojousima** [comment]
Continuing at #37 with these commits and some additional commits.

**Pull request closed without merging**